Source/SwiftLintFramework/Rules/FileHeaderRule.swift

```diff
@@ -83,7 +83,7 @@ public struct FileHeaderRule: ConfigurationProviderRule, OptInRule {
                 StyleViolation(
                     ruleDescription: type(of: self).description,
                     severity: configuration.severityConfiguration.severity,
-                    location: Location(file: file.path)
+                    location: Location(file: file.path, line: 1, character: 1) // Force to first line and character
```

Comment:
could you remove the `character` parameter since in this case the warning is not related to a particular offset?